storm-client/src/jvm/org/apache/storm/utils/CuratorUtils.java

```diff
@@ -75,29 +72,7 @@ public List<ACL> getAclForPath(String s) {
 
     protected static void setupBuilder(CuratorFrameworkFactory.Builder builder, final String zkStr, Map<String, Object> conf,
                                        ZookeeperAuthInfo auth) {
-        List<String> exhibitorServers = ObjectReader.getStrings(conf.get(Config.STORM_EXHIBITOR_SERVERS));
-        if (!exhibitorServers.isEmpty()) {
-            // use exhibitor servers
-            builder.ensembleProvider(new ExhibitorEnsembleProvider(
-                new Exhibitors(exhibitorServers,
-                        ObjectReader.getInt(conf.get(Config.STORM_EXHIBITOR_PORT)),
-                        new Exhibitors.BackupConnectionStringProvider() {
-                            @Override
-                            public String getBackupConnectionString() throws Exception {
-                                // use zk servers as backup if they exist
-                                return zkStr;
-                            }
-                        }),
-                new DefaultExhibitorRestClient(),
-                ObjectReader.getString(conf.get(Config.STORM_EXHIBITOR_URIPATH)),
```

Comment:
are these config options no longer used?  Can we remove them as well?

Reply:
Good catch! Will clean it up tmrw.

Reply:
fixed